Gemfile

```diff
@@ -1,3 +1,10 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
```

Comment:
I understand your idea. It is not possible to use the RAILS_GROUPS but I can move to a profile group and after use `Bundler.require(:profiling)` on the script and it will work beautifully. Is this ok for you?